src/pages/sitemap/sitemap-index.xml.ts

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+import path from 'path';
+import { getServerSideSitemap } from 'next-sitemap';
+import { GetServerSideProps } from 'next';
+import { getAllContentItems } from '../../service/get-all-content';
+import { getTopicPagePathMappings } from '../../service/get-topic-paths';
+
+const DEVCENTER_URL =
+    process.env.DEVCENTER_URL || 'https://mongodb.com/developer';
```

Comment:
Where is `process.env.DEVCENTER_URL` set? Do you mean `HOST_URL`?

Reply:
I didn't use `HOST_URL` because I wanted the sitemap to have the correct values locally. my understanding is that locally & in staging we'd want to see the production links and not the ones specific to that environment (e.g. `https://mongodb.com/developer/articles` instead of `localhost:3000/articles`)/

`DEVCENTER_URL` is just an unused env variable name that I came up with for an extra bit of configurability, but it probably isn't necessary as I'm assuming the URL to the site likely won't change, and if it does it can be changed in the code. I'm happy to remove this.